bigchaindb/common/transaction.py

```diff
@@ -410,7 +410,7 @@ class Transaction(object):
     TRANSFER = 'TRANSFER'
     GENESIS = 'GENESIS'
     ALLOWED_OPERATIONS = (CREATE, TRANSFER, GENESIS)
-    VERSION = bigchaindb.version.__version__
+    VERSION = bigchaindb.version.__short_version__[:-4]  # 0.9, 0.10 etc
```

Comment:
I don't understand how a test can resolve the concern of @sohkai 

`[:-4]` means, "Include all characters except for the last four."